src/jvm/main/org/jetbrains/kotlinx/lincheck/transformation/LincheckClassVisitor.kt

```diff
@@ -139,7 +139,7 @@ internal class LincheckClassVisitor(
         // We need to disable breakpoints in such a case, as the numeration will break.
         // Breakpoints are disabled as we do not instrument toString and enter an ignored section,
         // so there are no beforeEvents inside.
-        if (methodName == "<init>" || ideaPluginEnabled && methodName == "toString" && desc == "()Ljava/lang/String;") {
+        if (ideaPluginEnabled && methodName == "toString" && desc == "()Ljava/lang/String;") {
```

Comment:
Reminder: this part was extracted into separate PR https://github.com/JetBrains/lincheck/pull/661

Reply:
I know, I am waiting for its merge.